hudi-common/src/main/java/org/apache/hudi/common/model/WriteOperationType.java

```diff
@@ -68,6 +68,10 @@ public enum WriteOperationType {
    * Convert string value to WriteOperationType.
    */
   public static WriteOperationType fromValue(String value) {
+    if (value == null) {
+      // NOTE: Write operation type of commit metadata generated by Flink may be null for the version before 0.12.
```

Comment:
The write operation type of commit metadata generated by Flink isn't null after #8070. Therefore the write operation type of commit metadata is null before 0.12 version, which causes that `ConcurrentOperation#init` will throw `NullPointerException`. cc @danny0405.

Reply:
We should fix it to be never null.